risc0/build/src/docker.rs

```diff
@@ -76,21 +95,62 @@ pub fn docker_build(manifest_path: &Path, src_dir: &Path, features: &[String]) -
         let temp_path = temp_dir.path();
         let rel_manifest_path = manifest_path.strip_prefix(&src_dir)?;
         create_dockerfile(rel_manifest_path, temp_path, pkg_name.as_str(), features)?;
-        build(&src_dir, temp_path)?;
+        build(src_dir.as_path(), temp_path)?;
     }
     println!("ELFs ready at:");
 
-    let target_dir = src_dir.join(TARGET_DIR);
-    for target in root_pkg.targets.iter() {
+    for target in get_targets(&root_pkg) {
```

Comment:
Why did this change?

Reply:
Same as [here](https://github.com/risc0/risc0/pull/1592#discussion_r1544908955) - To avoid code duplication. 

I realize now it's better to just use `root_pkg` as it was before - so the changes were reverted ✅